src/event.js

```diff
@@ -511,10 +498,10 @@ jQuery.event = {
 // synthetic events by interrupting progress until reinvoked in response to
 // *native* events that it fires directly, ensuring that state changes have
 // already occurred before other listeners are invoked.
-function leverageNative( el, type, expectSync ) {
+function leverageNative( el, type, isSetup ) {
```

Comment:
@gibson042 This is a tricky function, I'd feel safer if you reviewed those changes.

Reply:
It’s much easier to understand the logic when asynchronous handling is removed from here, though. 😅

Reply:
This code landed separately, as part of making focus events sync in IE.